src/styles/main.scss

```diff
@@ -1,3 +1,29 @@
-body {
-  margin: 0;
+@import "blocks/normalize.scss";
+@import "blocks/variables.scss";
+@import "blocks/stars.scss";
+@import "blocks/header.scss";
+@import "blocks/navigation.scss";
+@import "blocks/card.scss";
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.container {
```

Comment:
something weird happens on lower resolutions, try to fix it:
<img width="780" alt="image" src="https://user-images.githubusercontent.com/37161315/157039812-a620a75d-f966-4fe0-9262-048f5f29c8f1.png">
<img width="431" alt="image" src="https://user-images.githubusercontent.com/37161315/157039838-76b39b9d-df0c-430b-80b1-b80df5e104c8.png">
<img width="335" alt="image" src="https://user-images.githubusercontent.com/37161315/157039851-f0f5569d-470a-4585-836c-77ff037bcb18.png">